pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -399,6 +399,29 @@ in {
     };
   };
 
+  octolapse = buildPlugin rec {
+    pname = "Octolapse";
+    version = "0.4.1";
+
+    src = fetchFromGitHub {
+      owner = "FormerLurker";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "13q20g7brabplc198jh67lk65rn140r8217iak9b2jy3in8fggv4";
+    };
+
+    pythonImportsCheck = [ "file_read_backwards" ];
```

Comment:
**TL;DR**: I think we'll have to give up on the import check in this plugin. I suggest you remove this import check entirely (read below for a more detailed explanation):
```suggestion
```

**Detailed explanation**:

Similarly to the other comment, this should be like this:
```nix
    pythonImportsCheck = [ "octoprint_octolapse" ];
```

However, I just checked and octoprint's plugins execute some code on import which fails the import check:
```
python3.8-OctoPrintPlugin-Octolapse> Check whether the following modules can be imported: octoprint_octolapse
python3.8-OctoPrintPlugin-Octolapse> Initializing GcodePositionProcessor V1.0.1 - Copyright (C) 2019  Brad Hochgesang...Python 3+ Detected...complete
python3.8-OctoPrintPlugin-Octolapse> 2021-11-27 16:07:30,253 - octolapse.__init__ - INFO - Release mode detected.
python3.8-OctoPrintPlugin-Octolapse> Error in atexit._run_exitfuncs:
python3.8-OctoPrintPlugin-Octolapse> Traceback (most recent call last):
python3.8-OctoPrintPlugin-Octolapse>   File "/nix/store/66p7i56lg3zzp7nj6g0fywb704ihp4iy-python3-3.8.11/lib/python3.8/logging/__init__.py", line 2127, in shutdown
python3.8-OctoPrintPlugin-Octolapse>     h.close()
python3.8-OctoPrintPlugin-Octolapse>   File "/nix/store/iqdafs15896fzih94jgicr7lgx0b4h5j-python3.8-OctoPrint-1.6.1/lib/python3.8/site-packages/octoprint/logging/handlers.py", line 31, in close
python3.8-OctoPrintPlugin-Octolapse>     self._executor.shutdown(wait=True)
python3.8-OctoPrintPlugin-Octolapse> AttributeError: 'OctolapseConsoleHandler' object has no attribute '_executor'
```

@SuperSandro2000: I suggest we omit the imports check here. It would require a change a change to the `buildPlugin` function to support this.

Reply:
Makes sense to me. If everything else looks good, I'll go ahead and **comment out the check**, it may be sensible to add it at a later time when the import works.

Reply:
Good to know.